pkg/test/env/environment.go

```diff
@@ -204,8 +204,13 @@ func (e *Environment) LoadFixtureBuilderFactories(factories ...fixtures.FixtureB
 		return nil
 	}
 
+	var err error
+	var fixtureBuilder fixtures.FixtureBuilder
```

Comment:
Nit: fixtureBUilder doesn't have to be scoped outside the for loop

Reply:
:+1: